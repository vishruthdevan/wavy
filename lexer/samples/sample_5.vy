aud1 = load("input1.wav")
format = "aac"

if (format == 'wav') {
    export(aud1, 'output1.wav')
} else if (format == 'mp3') {
    changeFormat(aud1, format: 'wav')
    export(aud1, 'output1.mp3')
} else if (format == 'aac') {
    export(aud1, 'output1.aac')
} else {
    print("Invalid format)
}