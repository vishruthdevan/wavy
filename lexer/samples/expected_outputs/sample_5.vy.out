<IDENTIFIER, "aud1">
<ASSIGN, "=">
<LOAD, "load">
<LPR, "(">
<STRING, "input1.wav">
<RPR, ")">
<IDENTIFIER, "format">
<ASSIGN, "=">
<STRING, "aac">
<IF, "if">
<LPR, "(">
<IDENTIFIER, "format">
<EQUALS, "==">
<STRING, "wav">
<RPR, ")">
<LBRACE, "{">
<EXPORT, "export">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<STRING, "output1.wav">
<RPR, ")">
<RBRACE, "}">
<ELSE, "else">
<IF, "if">
<LPR, "(">
<IDENTIFIER, "format">
<EQUALS, "==">
<STRING, "mp3">
<RPR, ")">
<LBRACE, "{">
<IDENTIFIER, "changeFormat">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<IDENTIFIER, "format">
<COLON, ":">
<STRING, "wav">
<RPR, ")">
<EXPORT, "export">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<STRING, "output1.mp3">
<RPR, ")">
<RBRACE, "}">
<ELSE, "else">
<IF, "if">
<LPR, "(">
<IDENTIFIER, "format">
<EQUALS, "==">
<STRING, "aac">
<RPR, ")">
<LBRACE, "{">
<EXPORT, "export">
<LPR, "(">
<IDENTIFIER, "aud1">
<COMMA, ",">
<STRING, "output1.aac">
<RPR, ")">
<RBRACE, "}">
<ELSE, "else">
<LBRACE, "{">
<IDENTIFIER, "print">
<LPR, "(">
<STRING, "Invalid format)
}">
<EOF, "">
